code/game/objects/items/crayons.dm

```diff
@@ -824,8 +824,8 @@
 		var/fraction = min(1, . / reagents.maximum_volume)
 		reagents.expose(carbon_target, VAPOR, fraction * volume_multiplier)
 
-	else if(actually_paints && target.atom_colours[atom_colours] == paint_color)
-		balloon_alert(user, "[target.p_theyre()] already of that color.")
+	else if(actually_paints && (target.color == paint_color || target.atom_colours?[WASHABLE_COLOUR_PRIORITY] == paint_color))
```

Comment:
we should have a getter on atom_colours for the highest priority below some threshold color
this seems a bit odd